djangocms_versioning/helpers.py

```diff
@@ -300,16 +299,26 @@ def remove_published_where(queryset):
     return queryset
 
 
-# FIXME: This should reuse a generic method that uses the groupers defined filters
-def get_latest_admin_viewable_page_content(page, language):
+def get_latest_admin_viewable_content(grouper, include_unpublished_archived=False, **extra_grouping_fields):
```

Comment:
Does this have any unit tests?

Reply:
It now has here: https://github.com/django-cms/djangocms-versioning/blob/edfb62c8202105f3a37c27aa7e2f244567f0d7f9/tests/test_indicators.py#L18-L30